### DISCUSSION TITLE
[Esame Ottobre 2023] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#374] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/3482a173-78e1-4b64-a62f-2b00be7b97c6)

**Comment by Elia-Belli** [comment]
**(a)** Il sistema è compatibile perchè $(18,8)=2|2$, $(21,9)=3|12$ e $(22,14)=2|10$ ed equivalente ad un cinese

$$\begin{cases}4x\equiv 1(9)\\
3x\equiv 4(7)\\
7x\equiv 5(11)\end{cases}\to\begin{cases}x\equiv 7(9)\\
x\equiv 6(7)\\
x\equiv 7(11)\end{cases}$$

Procedendo col teorema cinese del resto ottengo $x\equiv 601(693)$

**Comment by Frisbon** [comment]
> as always tag if im wrong (corretto, ty @Traster-uni)
![image_2024-01-27_09-28-21](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/52c0ddd9-43be-45d1-834b-6f1b305848f6)

**Reply by Traster-uni** [reply]
Ho l'impressione che al sistema b stanno degli errori:
    -18t = 18 (20) => t = 2 (20) perché 9t=81(10) in quanto 1=9(9)+10(-9)
             poi t=1(10) che moltiplicato nuovamente per 2 viene t=2(20)
    -18 * 20 = 360 non 106, che in mod 24 e' 0 poiche' 360 = 24(15)+0.
    -5+18 = 23 che in mod 24 e' -1
